test/e2e/common_test.go

```diff
@@ -498,6 +514,8 @@ func (p *PodmanTestIntegration) PodmanPID(args []string) (*PodmanSessionIntegrat
 
 // Cleanup cleans up the temporary store
 func (p *PodmanTestIntegration) Cleanup() {
+	se := SystemExec("iptables", []string{"-t", "nat", "-nvL"})
+	fmt.Println(se)
```

Comment:
Is this debugging?

Reply:
yup, for now. id like to leave it for a while until everything is completely settled down.  If folks dont like it, I will pull it however.